draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -199,8 +183,7 @@ When there is no user consent, the following filtering should be done to prevent
 
 3. SDP does not expose any a=candidate line corresponding to a host ICE candidate which contains an IP address.
 
-4. RTCIceCandidateStats dictionaries exposed to web pages do not contain any 'ip' member if related to a host ICE candidate.
-This also applies to any remote candidate in list-B.
+4. RTCIceCandidateStats dictionaries exposed to web pages do not contain any 'address' member if related to a host ICE candidate or a peer-reflexive remote candidate.
```

Comment:
This line is specifically tailored for the W3C WebRTC spec. It might be a good idea to phrase this in a less implementation-specific way to include non-browser implementations or potential alternative APIs. Something like...

> Statistics related to ICE candidates shall not contain the resolved IP address of a local host candidate or a peer-reflexive remote candidate.

Reply:
Agreed that we should try to push this wording in the webrtc stats spec.
@lgrahl suggestion plus a direct reference to RTCIceCandidateStats dictionaries seems fine in the meantime.
For instance:
Statistics related to ICE candidates SHOULD NOT expose the resolved IP addresses of a local host candidate or a peer-reflexive remote candidate. This applies in particular to the 'address' member of RTCIceCandidateStats dictionaries.

Reply:
Since we now have a section above that talks about this problem, I think we should just move this text directly to that section.